Flutter/calendar/drag-drop.md

```diff
@@ -125,7 +125,7 @@ void dragUpdate(AppointmentDragUpdateDetails appointmentDragUpdateDetails) {
 [targetResource](https://pub.dev/documentation/syncfusion_flutter_calendar/latest/calendar/AppointmentDragEndDetails/targetResource.html) - Get the target resource details.
 
 {% tabs %}
-{% highlight Dart %}
+{% highlight dart hl_lines="9 10" %}
```

Comment:
Check 9 is needed or not and the highlight method also

Reply:
Method highlighted.